test/cov.jl

```diff
@@ -1,6 +1,7 @@
 using StatsBase
 using Base.Test
 
+@testset "StatsBase.Covariance" begin
```

Comment:
I copied the naming convention from [test/hist.jl](https://github.com/JuliaStats/StatsBase.jl/blob/master/test/hist.jl), but I imagine once all the tests have been converted to using `@testset` this should be changed.